wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -1114,4 +1117,15 @@ public void checkSendingConditions(
                 crowdNodeConfig
         );
     }
+
+    public final HashCode apkHash() throws IOException {
```

Comment:
This should be moved to `PackageInfoProvider`, and then `getPackageCodePath()` can be accessed from the `@ApplicationContext private val context: Context`

Reply:
These changes were made.